int64 graspID
---
sensor_msgs/PointCloud2 graspPtCld